shellfoundry/commands/pack_command.py

```diff
@@ -12,18 +14,27 @@ def __init__(self):
         self.config_reader = ShellConfigReader()
         self.package_builder = PackageBuilder()
         self.dependencies_packager = PythonDependenciesPackager()
+        self.shell_package_builder = ShellPackageBuilder()
 
     def pack(self):
+
+        current_path = os.getcwd()
+
+        if self.shell_package_builder.is_tosca_based_shell(current_path):
+            self.shell_package_builder.pack()
```

Comment:
@doppleware we never send the current path to the command in order to keep all the logic inside the commands/utilities and not in bootstrap.py